homeassistant/components/sensibo/coordinator.py

```diff
@@ -94,6 +94,33 @@ async def _async_update_data(self) -> dict[str, dict[str, Any]]:
             calibration_temp = dev["sensorsCalibration"].get("temperature", 0.0)
             calibration_hum = dev["sensorsCalibration"].get("humidity", 0.0)
 
+            # Sky plus supports functionality to use motion sensor as sensor for temp and humidity
+            if dev["mainMeasurementsSensor"]:
+                temperature = dev["mainMeasurementsSensor"]["measurements"].get(
+                    "temperature", 0.0
+                )
+                humidity = dev["mainMeasurementsSensor"]["measurements"].get(
+                    "humidity", 0
+                )
```

Comment:
```suggestion
            if main_sensor := dev["mainMeasurementsSensor"]:
                measurements = main_sensor["measurements"]
                temperature = measurements.get("temperature")
                humidity = measurements.get("humidity")
```

Usually we use `None` to represent an unknown value.  0 is possibly a valid value here so we probably shouldn't use it.

Reply:
I would say that makes sense